modules/graphql_views/src/Plugin/Deriver/ViewDeriver.php

```diff
@@ -96,7 +96,7 @@ public function getDerivativeDefinitions($basePluginDefinition) {
         $this->derivatives[$id] = [
           'id' => $id,
           'name' => graphql_core_propcase($name) . 'View',
-          'type' => graphql_core_camelcase($type),
+          'type' => "Entity",
```

Comment:
We could add a schema building step to eliminate fields that have missing types.